internal/slicer/slicer.go

```diff
@@ -153,11 +153,13 @@ func Run(options *RunOptions) error {
 		if reader == nil {
 			continue
 		}
+		fsCreator := fsutil.NewCreator()
```

Comment:
Do we need a different `fsCreator` for every slice?

Reply:
Yes. That is one of the bits that can no longer be seen in this trimmed down version of the PR. We need to create one for each because we want to report the files created for each one. If we used the same `fsCreator` we would not know which slice created which file (unless we resorted to emptying the map or some other hack).